ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_internal_order.schema.yml

```
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  vlan_internal_order:
+    type: dict
+    display_name: Internal VLAN Order
+    keys:
+      allocation:
+        type: str
+        required: true
+        valid_values: ["ascending", "descending"]
+      range:
+        type: dict
+        required: true
+        keys:
+          beginning:
+            type: int
```

Let us add some extra checks since this is a Vlan ID.
```suggestion
            type: int
            min: 1
            max: 4094
```